docs/benchmarks.md

```diff
@@ -11,6 +11,18 @@ void my_benchmark(nvbench::state& state) {
 NVBENCH_BENCH(my_benchmark);
 ```
 
+The following example shows how to benchmark functions that do not expose stream parameters:
+```cpp
+void my_benchmark(nvbench::state& state) {
+  state.set_cuda_stream(nvbench::cuda_stream{cudaStreamDefault, false});
```

Comment:
Even though the value of `cudaStreamDefault` is 0, it isn't intended to be a `cudaStream_t`. It's an argument to [`cudaStreamCreateWithFlags`](https://docs.nvidia.com/cuda/cuda-runtime-api/group__CUDART__STREAM.html#group__CUDART__STREAM_1gb1e32aff9f59119e4d0a9858991c4ad3). 
```suggestion
  cudaStream_t default_stream = 0;
  state.set_cuda_stream(nvbench::cuda_stream{default_stream, false});
```